src/lwc/gsResources/gsResources.js-meta.xml

```diff
@@ -4,6 +4,11 @@
     <isExposed>true</isExposed>
     <masterLabel>NPSP Get Started Page Resources</masterLabel>
     <targets>
-          <target>lightning__AppPage</target>
+        <target>lightning__AppPage</target>
     </targets>
+    <targetConfigs>
+        <targetConfig targets="lightning__AppPage">
+            <property name="belongsTo" label="Belongs to" type="String" datasource="Admin,End User" />
```

Comment:
```suggestion
            <property name="pageType" label="Page Type" type="String" datasource="Admin,End User" />
```

Reply:
I agree with Mike's suggestion to change the field name.